src/main/java/racing/Racing.java

```diff
@@ -5,37 +5,59 @@
 
 import static racing.InputView.*;
 import static racing.ResultView.*;
+import static racing.Validate.*;
 
 public class Racing {
 
     public static void main(String[] args) {
-        int carCount = inputCarCount();
+        String carNames = inputCarCount();
 
-        List<Car> carList = new ArrayList<>();
-        for(int i = 0; i < carCount; ++i){
-            carList.add(new Car());
+        List<Car> cars = new ArrayList<>();
+        for(String name : carNames.split(",")){
+            validName(name);
+            cars.add(new Car(name));
         }
         int moveCount = inputMoveCount();
 
         System.out.println("실행 결과");
-        doRacing(carList, moveCount);
+        doRacing(cars, moveCount);
+
+        showResult(getWinners(cars));
     }
 
-    public static void doRacing(List<Car> carList, int moveCount) {
+    public static void doRacing(List<Car> cars, int moveCount) {
+        Strategy strategy = new RandomStrategy();
+
         for(int i = 0; i < moveCount; ++i){
-            moveCars(carList);
+            moveCars(cars, strategy);
+            showStatus(cars);
         }
-
-        showResult(carList);
     }
 
-    private static void moveCars(List<Car> carList) {
-        Strategy strategy = new RandomStrategy();
-        
-        for(Car car : carList){
+    private static void moveCars(List<Car> cars, Strategy strategy) {
+        for(Car car : cars){
```

Comment:
```suggestion
        for (Car car : cars) {
```
이렇게 띄어쓰기를 사용하는 것이 좋습니다.
보편적인 컨벤션입니다 😄